src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -58,12 +96,98 @@
                 [:div {:style style :onClick #(add-crumb :workspace-import "Choose Workspace")}
                  "Copy from another workspace"]]))]])}])})
 
+(react/defc EntityAttributes
+  {:render
+   (fn [{:keys [state props]}]
+     (let [entity-type (:entity-type props)
+           entity-name (:entity-name props)
+           attributes (:attr-list props)
+           item-column-name (get-column-name entity-type)
+           setColumns [{:header item-column-name :starting-width 320 :sort-initial :asc :sort-by :text
+                        :as-text (fn [x] (:entityName x)) :content-renderer (fn [x] x)}]
+           singleColumns [{:header "Attribute" :starting-width 120 :sort-initial :asc}
+                          {:header "Value" :starting-width :remaining
+                           :content-renderer (fn [attr-value]
+                                               (if-let [parsed (common/parse-gcs-uri attr-value)]
+                                                 [GCSFilePreviewLink (assoc parsed
+                                                                       :attributes {:style {:display "inline"}}
+                                                                       :link-label attr-value)]
+                                                 attr-value))}]]
+       [:div {:style {:flexBasis (if attributes "20%" 0) :ref "entity-attributes-list"}}
+        [:div {:style {:marginLeft ".38em"}}
+         (when (some? attributes)
+           [:div {:style {:fontWeight "bold" :padding "0.7em 0" :marginBottom "1em"}}
+            (if (= item-column-name "Entity") (str entity-name "  Attributes:")
+                                              (str entity-name "  " item-column-name "s:"))])
+         (when (some? attributes)
+           [:div {}
+            [table/Table {:reorderable-columns? false
+                          :width :narrow
+                          :pagination :none
+                          :filterable? false
+                          :initial-rows-per-page 100
+                          :always-sort? true
+                          :header-row-style {:borderBottom (str "2px solid " (:line-default style/colors))
+                                             :backgroundColor "white" :color "black" :fontWeight "bold"}
+                          :empty-message (if (= item-column-name "Entity") (str "No Entity Attributes defined")
+                                                                           (str "No " item-column-name "s defined"))
+                          :columns (if (is-entity-set? entity-type) setColumns singleColumns)
+                          :data (seq attributes)
+                          :->row
+                          (fn [x]
+                            (if (map? x)   ;;map is a sample_set or pair_set or participant_set
```

Comment:
Nit: The convention is usually single-semicolon comments for those that appear on the same line as code, double-semicolon for comments that are alone on a line.